README.md

```diff
@@ -18,6 +18,7 @@
 - [fzf](https://github.com/junegunn/fzf)
 - [Node.js](https://nodejs.org/en/download/)
 - [ffmpeg](https://ffmpeg.org/download.html)
+- [openssl](https://www.openssl.org/source/)
```

Comment:
- Please mark it as optional for parallel jobs/threads
- Please add xxd as dependency

Reply:
Resolved:
 * Moved Option Dependencies to another subsection

```
### Optional Dependencies for parallel operations
- [openssl](https://www.openssl.org/source/)
- [xxd] (https://github.com/vim/vim/tree/master/src/xxd)
```